server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/eq/EarthquakeModel.java

```diff
@@ -23,19 +23,19 @@ public class EarthquakeModel extends Earthquake {
     private final String siteAmplification;
 
     // Visualization raster
-    private HazardDataset rasterDataset;
+    private HazardDataset hazardDataset;
```

Comment:
Have we decided to make it `HazardDataset[] hazardDatasets`?

Reply:
Yeah, I did that now